webknossos-datastore/app/com/scalableminds/webknossos/datastore/storage/RedisTemporaryStore.scala

```diff
@@ -59,9 +59,9 @@ trait RedisTemporaryStore extends LazyLogging {
       r.exists(id)
     }
 
-  def remove(id: String): Fox[Unit] =
+  def remove(id: String): Fox[Boolean] =
```

Comment:
Is this returned boolean used anywhere? I didn’t find a usage, and if not, I’d rather have it be Unit again